boards/arm/lpcxpresso55s69/lpcxpresso55s69.dtsi

```diff
@@ -111,4 +111,9 @@ arduino_gpio: &gpio1 {
 };
 
 mikrobus_spi: &hs_lspi {
+	eth_click: eth_click@1 {
+		compatible = "microchip,enc28j60";
+		reg = <1>;
+		status = "disabled";
+	};
```

Comment:
Not all LPCXpresso55S69 boards have this shield installed. Why add it here?

Reply:
I've disabled it by default. This is defined here for `reg = <1>` which tells the CS 1 is used by SPI.

Reply:
That is not how our shield support is supposed to work (imagine having to add all shields to all boards in a disabled state). Please see https://docs.zephyrproject.org/latest/guides/porting/shields.html for further details.

Reply:
If you need board specific shield configuration, you can add it under `boards/shields/mikro_.../boards/lpcxpresso.../`

Reply:
@erwango i'm not happy with this solution, SPI's CS in use for this board's MikroBus is #1, it should be specified at board level, so we don't have to mention `reg = <1>` on all shields that may go onto this board.

Reply:
@agansari, I'm not sure to get your point, but as mentioned by @henrikbrixandersen this is not the way shields are supposed to work and this solution doesn't scale.